recipe/meta.yaml

```diff
@@ -0,0 +1,248 @@
+{% set version = "3.0.0" %}
+{% set build_ext_version = version %}
+{% set build_ext = "cuda" if build_type == 'cuda' else "cpu" %}
+{% set proc_build_number = "1" %}
+{% set checksum = "73c2cc3be537aa1f3fd9490cfec185714168c9bfd599d23e287ab0cc0558e27a" %}
+
+package:
+  name: arrow-cpp-ext
+  version: {{ version }}
+
+source:
+  url: https://dist.apache.org/repos/dist/release/arrow/arrow-{{ version }}/apache-arrow-{{ version }}.tar.gz
+  sha256: {{ checksum }}
+
+build:
+  number: 4
+  run_exports:
+    - {{ pin_subpackage("arrow-cpp", max_pin="x.x.x") }}
+
+outputs:
+  - name: arrow-cpp-proc
+    version: {{ build_ext_version }}
+    build:
+      number: {{ proc_build_number }}
+      string: {{ build_ext }}
```

Comment:
I think you need to add `{{ PKG_BUILDNUM }}` to the build string here.

Reply:
I don't expect it'll ever need to change (content-wise, or build number-wise) but yeah, i can add that.